prime-dotnet-webapi/Models/SiteRegistration/Site.cs

```diff
@@ -16,7 +16,6 @@ public class Site : BaseAuditable
 
         public int OrganizationId { get; set; }
 
-        [JsonIgnore]
```

Comment:
Is this intentional?

Reply:
Yeah I needed it to display the organization name in the list of sites